examples/machine-learning-pipeline/README.md

```diff
@@ -859,9 +910,32 @@ You can read more about the details of the pipeline components on the [pipeline'
 Before you start, make sure you have your personal git access token ready. The git menu option on the left bar of the workbench requires the personal token to connect to git and clone the repo.
 Also make sure to have a gcs bucket ready to store the artifacts for the tutorial. To deploy the bucket, you can go to service catalog and create a new deployment from the storage bucket solution.
 
+Additionally, the following Service Accounts need to be created with the respective roles since the Compute Engine SA cannot to be used to deploy the Dataflow and Vertex Pipeline steps:
+
+`dataflow_runner_sa@prj-d-bu3machine-learning-[project-number].iam.gserviceaccount.com`
+
+This service account requires the following roles:
+* `roles/bigquery.admin`
+* `roles/dataflow.admin`
+* `roles/dataflow.worker`
+* `roles/storage.admin`
+
+
+`vertex_model_sa@prj-d-bu3machine-learning-[project-number].iam.gserviceaccount.com`
+
+No role is required for the `vertex_model_sa`.
+
+Run the command below to grant the notebook to be able to create jobs in BigQuery:
+```
+bq query --nouse_legacy_sql \
+'ALTER PROJECT `prj-d-bu3machine-learning-[project-number]` SET OPTIONS \
+(`region-us-central1.default_kms_key_name`="projects/[prj-d-kms-project-ID]/locations/us-central1/keyRings/sample-keyring/cryptoKeys/prj-d-bu3machine-learning");'
+```
+
```

Comment:
Don't we need another one? the SA to run the pipeline. Then roles to act as service account

Reply:
As far as I remember, this was the command I ran to have the BQ jobs working. I will add the details about the SA to run the workbench.